src/werkzeug/datastructures.py

```diff
@@ -355,6 +355,15 @@ def __setstate__(self, value):
         dict.clear(self)
         dict.update(self, value)
 
+    def __iter__(self):
+        """This looks like a no-op, but actually preserves required interop.
+
+        See https://github.com/pallets/werkzeug/pull/2043.
```

Comment:
Can we use a direct reference to the PR here? I think we are using the sphinx-issues plugin

    :pr:`2043`